packages/fether-electron/package.json

```diff
@@ -51,7 +51,7 @@
   "devDependencies": {
     "copyfiles": "^2.0.0",
     "cross-env": "^5.2.0",
-    "electron": "^2.0.2",
+    "electron": "^3.1.0",
```

Comment:
Why not 4, since it's also out?

Reply:
I tried Electron 4 a couple of weeks ago, and didn't have any luck with that either. https://github.com/paritytech/fether/compare/luke-309-electron-4?expand=1